biggraphite/carbon_plugin.py

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env python
```

Comment:
Move to biggraphite/plugins/carbon.py

It will be prettier in the config file: DATABASE_PLUGIN = 'biggraphite.plugins.carbon.BigGraphiteDatabase'